src/lambdas/CompleteFileUpload/test/index.test.ts

```diff
@@ -1,10 +1,10 @@
 import * as sinon from 'sinon'
-import * as DynamoDB from '../../../lib/vendor/AWS/DynamoDB.js'
-import * as S3 from '../../../lib/vendor/AWS/S3.js'
-import {getFixture} from '../../../util/mocha-setup.js'
-import {handler} from '../src/index.js'
+import * as DynamoDB from '../../../lib/vendor/AWS/DynamoDB'
```

Comment:
wait didn't the last commit add all these ".js" suffixes? are we just removing them again?